scripts/sign_all.py

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I'd remove it in a separate commit/pull request (that we can merge before this one)